test_to_do_app/test_to_do_app.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime,timedelta
+from itertools import count
```

Comment:
It looks like this import (`from itertools import count`) is unused - possibly an accidental import when trying to find the `len` function? We should clear this up just to save future developers from being confused